plugin-server/tests/worker/ingestion/hooks.test.ts

```diff
@@ -101,9 +101,41 @@ describe('hooks', () => {
     describe('getValueOfToken', () => {
         const action = { id: 1, name: 'action1' } as Action
         const event = { distinct_id: 2, properties: { $browser: 'Chrome' } } as unknown as PluginEvent
-        const person = {} as Person
+        const person = { properties: { enjoys_broccoli_on_pizza: false } } as unknown as Person
+
+        test('person name', () => {
+            const tokenUserName = ['person', 'name']
+
+            const [text, markdown] = getValueOfToken(
+                action,
+                event,
+                person,
+                'http://localhost:8000',
+                WebhookType.Teams,
+                tokenUserName
+            )
+
+            expect(text).toBe('2')
```

Comment:
it's a bit odd that we ask for a person "name" and we get "2" isn't it? Maybe we shouldn't make it be `person.name` but `person.username` not sure?

Reply:
Yeah, it isn't actually the "name". Though "username" would also be misleading. Actually I think this behavior is closest to Python's `__str__` method – it's just an implicit implementation-defined way of stringifying the object – so replaced `[person.name]` with `[person]`. This should avoid raised eyebrows.